core/templates/pages/about-page/about-page.component.ts

```diff
@@ -19,100 +19,79 @@
 import { Component, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 
-import { AboutPageConstants } from './about-page.constants';
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service.ts';
 import { WindowRef } from
   'services/contextual/window-ref.service.ts';
-
-interface CreditNames {
-  letter: string;
-  names: string[];
-}
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+import { UserService } from 'services/user.service';
+import { LoaderService } from 'services/loader.service.ts';
+import constants from 'assets/constants';
```

Comment:
Add all the above imports in alphabetically sorted order*

Reply:
done